owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/shares/RemoteShare.kt

```diff
@@ -42,8 +42,8 @@ data class RemoteShare(
     var sharedWithAdditionalInfo: String = "",
     var name: String = "",
     var shareLink: String = "",
-    var fileSource: Long = 0,
-    var itemSource: Long = 0,
+    var fileSource: String = "0",
```

Comment:
Why "0" and not ""?

Reply:
Previously it was 0, so i kept the same value but as String. We are not using them at all, so this is just symbolic..